extensions/amp-story-panning-media/0.1/amp-story-panning-media.css

```diff
@@ -19,6 +19,8 @@ amp-story-panning-media {
   display: flex !important;
   justify-content: center !important;
   align-items: center !important;
+  /* For the translate3d z calculation. */
```

Comment:
Can you explain a bit more in the code comment why this is needed and how it works?